packages/main/src/index.ts

```diff
@@ -128,6 +129,28 @@ app.on('web-contents-created', (_event, contents) => {
     }
   });
 
+  /**
+   * Verify webview options before creation
+   *
+   * Strip away preload scripts, disable Node.js integration, and ensure origins are on the allowlist.
+   *
+   * @see https://www.electronjs.org/docs/latest/tutorial/security#12-verify-webview-options-before-creation
+   */
+  contents.on('will-attach-webview', (event, webPreferences, params) => {
+    delete webPreferences.preload;
+    // eslint-disable-next-line
+    // @ts-ignore
```

Comment:
Please don't use `@ts-ignore`. Use `@ts-expect-error Explainer WHY this error is expected` instead